modules/aws/iam.go

```diff
@@ -133,6 +133,50 @@ func EnableMfaDeviceE(t *testing.T, iamClient *iam.IAM, mfaDevice *iam.VirtualMF
 	return nil
 }
 
+// AssertIamPolicyExists checks if the given IAM policy exists in the given region and fail the test if it does not.
+func AssertIamPolicyExists(t *testing.T, region string, policyARN string) {
+	err := AssertIamPolicyExistsE(t, region, policyARN)
+	if err != nil {
+		t.Fatal(err)
+	}
+}
+
+//AssertIamPolicyExistsE checks if the given IAM policy exists in the given region and return an error if it does not.
+func AssertIamPolicyExistsE(t *testing.T, region string, policyARN string) error {
+	iamClient, err := NewIamClientE(t, region)
+	if err != nil {
+		return err
+	}
+
+	_, err = iamClient.GetPolicyVersion(&iam.GetPolicyVersionInput{
+		PolicyArn: &policyARN,
+	})
+	return err
```

Comment:
Instead of duplicating this, couldn't this method call `GetIamPolicyDocumentE` and discard the contents? Same for the S3 methods.